src/screens/ProductDetails/ProductDetails.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Screen from '../../styledComponents/Screen'
+import Button from '../../styledComponents/Button'
+import products from '../../data/products'
+import categories from '../../data/categories'
+import {connect} from 'react-redux'
+import { Link } from 'react-router-dom'
+import {addToCart} from '../../screens/ShoppingCart/actionCreators'
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    addToCart: (product,amount) => dispatch(addToCart(product,amount))
```

Comment:
I made it because I was thinking I will use it right after but then this feature goes for 'later' ;) I will use it in next update.